Source/SpatialGDK/Private/SpatialNetDriver.cpp

```diff
@@ -169,6 +169,44 @@ bool USpatialNetDriver::IsLevelInitializedForActor(const AActor* InActor, const
 	return true;
 }
 
+
+void USpatialNetDriver::NotifyActorDestroyed(AActor* ThisActor, bool IsSeamlessTravel /*= false*/)
+{
+	// Remove the actor from the property tracker map
+	RepChangedPropertyTrackerMap.Remove(ThisActor);
+
+	FActorDestructionInfo* DestructionInfo = NULL;
```

Comment:
`= nullptr`

Reply:
I see that a lot of this comes from the parent function. Still a good idea to make it proper.